### ISSUE TITLE
accuracy is always 1.0

### ISSUE THREAD

**Issue by tearhupo121031** [jacobgil/pytorch-pruning#18]
I run python finetune.py --train
the output accuracy is always 0
I use the dataset Dogs vs. Cats

**Comment by wltn0021** [comment]
Is there anyone who can verify and change labels?
I have same value.

**Comment by wltn0021** [comment]
Does anyone has this submission of test dataset?
Kaggle competition is over, so I can't verify which image is cat or dog
